packages/core/botpress/src/web/views/Logs/index.jsx

```diff
@@ -92,6 +92,21 @@ class LoggerView extends Component {
     return this.state.logs.filter(x => _.isString(x.message)).map(this.renderLine)
   }
 
+  downloadArchive = () => {
+    axios({
+      url: this.state.archiveUrl,
+      method: 'GET',
+      responseType: 'blob'
+    }).then(response => {
+      const url = window.URL.createObjectURL(new Blob([response.data]))
+      const link = document.createElement('a')
+      link.href = url
+      link.setAttribute('download', 'logs.txt')
+      document.body.appendChild(link)
+      link.click()
```

Comment:
@emirotin , the same is used in QNA `views/index.js`. Do you think it makes sense to extract it into helper?

Reply:
I think it's fine with replication for now but could be refactored later if we hit this place again.